### ISSUE TITLE
RequestLogFactory custom logging format

### ISSUE THREAD

**Issue by pandaadb** [dropwizard/dropwizard#1472]
Hi,

it appears to me, that the RequestLogFactory ignored all log formats that are passed in the configuration. It also hardcodes the RequestLogFactory as a layout. I know one can overwrite this factory and create a custom logger, however it would be nice if this could be done through configuration. 

Thanks,
Artur

**Comment by anev** [comment]
Hi,
I think this is done on purpose, take a look at the "Log Format" chapter in the manual:

```
Dropwizard’s log format has a few specific goals:

Be human readable.
Be machine parsable.
Be easy for sleepy ops folks to figure out why things are pear-shaped at 3:30 AM using standard UNIXy tools like tail and grep.
```

**Comment by pandaadb** [comment]
That is fine, but you can define your own log formats for your other loggers. Why not be able to have your own log format for request logging?

**Comment by ghenkes** [comment]
@pandaadb https://github.com/dropwizard/dropwizard/pull/1415 adds the ability to configure the format of the request log. It is available in master and targeted for the 1.0.0 release.

**Comment by nickbabcock** [comment]
Closing this because the feature is implemented in 1.0 (if this is not the case, or there is a bug, please report back to open the issue back up).
